tree/dataframe/inc/ROOT/RDFNodes.hxx

```diff
@@ -844,6 +860,12 @@ public:
       if (fNChildren == 1)
          fPrevData.IncrChildrenCount();
    }
+
+   /// This function must be defined by all nodes, but only the filters will add their name
+   void AddFilterName(std::vector<std::string> &filters){
```

Comment:
coding conventions require the `{` to be on a newline, maybe give a pass of clang-format to your changes (and only to your changes :smile: )

Reply:
Sometimes git-clang-format just does not do what it should :-1: I'll call it manually